src/clients/kol.ts

```diff
@@ -56,6 +56,16 @@ type PartialPlayer = {
   class: string;
 };
 
+type FullPlayer = {
```

Comment:
I'd probably like for this to extend PartialPlayer

Reply:
done, let me know if that suits

Reply:
Unless the code has loaded incorrectly you've done this in an odd way that I wasn't really aware was even possible